tools/install_reactive-resume.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+#function check_internet() {
+#  printf "Checking if you are online..."
+#  wget -q --spider http://github.com
+#  if [ $? -eq 0 ]; then
+#    echo "Online. Continuing."
+#  else
+#    error "Offline. Go connect to the internet then run the script again."
+#  fi
+#}
+
+#check_internet
```

Comment:
Why you've disabled internet check if you're using `wget`?

Reply:
should we change to curl? as i dont really sure wget installed by default.